packages/module-federation/src/utils/get-remotes-for-host.ts

```diff
@@ -59,7 +59,9 @@ function collectRemoteProjects(
   collected.add(remote);
 
   const remoteProjectRoot = remoteProject.root;
-  const remoteProjectTsConfig = remoteProject.targets['build'].options.tsConfig;
+  const remoteProjectTsConfig =
+    remoteProject.targets['build'].options.tsConfig ??
+    join(remoteProjectRoot, 'tsconfig.app.json');
```

Comment:
I know we generate with `tsconfig.app.json`. Technically users can change this setup.

Do we want some fallback here? It looks like it's needed to transpile `module-federation.config.ts`, so we could check `tsconfig.app.json`, `tsconfig.json`, and then the root `tsconfig.json`?

Reply:
Yes, very good point. I’ll add that